index.bs

```diff
@@ -1947,6 +1973,106 @@ This
 {{CredentialsContainer/get()|navigator.credentials.get()}} operation can be aborted by leveraging the {{AbortController}};
 see [[dom#abortcontroller-api-integration]] for detailed instructions.
 
+#### <dfn for="PublicKeyCredential" algorithm="Issuing a request to an authenticator">Issuing a request to an authenticator</dfn> #### {#sctn-issuing-request-to-authenticator}
```

Comment:
> I'm also thinking that this alg actually ought to follow [[DiscoverFromExternalSource]] rather than precede it, because the opening paragraphs of section 5.1.4 serve to introduce [[DiscoverFromExternalSource]] per se

[[DiscoverFromExternalSource]] calls this algorithm though, so maybe that's an argument to keep the order like this?

I added an introductory paragraph.

Reply:
> [[DiscoverFromExternalSource]] calls this algorithm though, so maybe that's an argument to keep the order like this?

I suppose. I'm just thinking from perspective of reader who reads the section 5.1.4 intro text, which intros `[[DiscoverFromExternalSource]]()`, but then the next subsection is an internal function of the latter.  I suppose with all the hyperlinks that it does not matter terribly.